mentorApp/lib/main.dart

```diff
@@ -16,7 +18,9 @@ class App extends StatelessWidget {
       '/': (context) => FirstRoute(),
       // When navigating to the "/second" route, build the SecondScreen widget.
       '/second': (context) => SecondRoute(),
-      '/UserProfile': (context) => AppController(),
+      '/filters': (context) => DropDownPage(),
```

Comment:
I think you don't need to add all the new pages to the `main.dart` file. I think it's handy to add routes when a workflow is not linear and can possibly load up different pages. However, since we know that once a sign up is created, you go from `Sign up -> Build profile -> Filters`, it's a straight path and we can directly call the widgets in the appropriate files.